sdk/communication/Azure.Communication.JobRouter/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,91 @@
+# Azure.Communication.JobRouter Samples - Hello World (sync)
+
+## Import the namespaces
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_UsingStatements
+using Azure.Communication.JobRouter;
+using Azure.Communication.JobRouter.Models;
+```
+
+## Create a client
+
+Create a `RouterClient` and send a request.
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_CreateClient
+var routerClient = new RouterClient(Environment.GetEnvironmentVariable("AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING"));
+```
+
+## Create a Distribution Policy
+
+Use `RouterClient` to create a [Distribution Policy](https://docs.microsoft.com/en-us/azure/communication-services/concepts/router/concepts#distribution-policy) to control how jobs are to be distributed to workers with associated queue.
+
+For this example, we are going to create a __Longest Idle__ policy with an offer TTL set to 30 seconds.
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_CreateDistributionPolicyLongestIdleTTL30S
+var distributionPolicy = routerClient.SetDistributionPolicy(
+    id: "distribution-policy-1",
+    name: "My Distribution Policy",
+    offerTTL: TimeSpan.FromSeconds(30),
```

Comment:
Maybe, for the first sample we should have a longer offer TTL than 3 seconds, so developers experimenting with it have a chance to see the offer?

Reply:
changed to have TTL of 1Day